Cilium released a new version, bpf.vlanBypass type changed

action url: https://github.com/spidernet-io/spiderpool/actions/runs/10083260034

```
 ===> Start to install cilium 
CILIUM_HELM_OPTIONS:  --set cni.exclusive=false                             --set kubeProxyReplacement=strict                             --set k8sServiceHost=spiderpool0724202136-control-plane                             --set k8sServicePort=6443                             --set bpf.vlanBypass=0  --set ipam.operator.clusterPoolIPv4PodCIDRList=10.244.64.0/18                                    --set ipam.operator.clusterPoolIPv6PodCIDRList=fd00:10:244::/112                                    --set ipv4.enabled=true                                    --set ipv6.enabled=true        --set image.repository=quay.io/cilium/cilium       --set image.useDigest=false       --set certgen.image.repository=quay.io/cilium/certgen       --set hubble.relay.image.repository=quay.io/cilium/hubble-relay       --set hubble.relay.image.useDigest=false       --set hubble.ui.backend.image.repository=quay.io/cilium/hubble-ui-backend       --set hubble.ui.frontend.image.repository=quay.io/cilium/hubble-ui       --set etcd.image.repository=quay.io/cilium/cilium-etcd-operator       --set operator.image.repository=quay.io/cilium/operator        --set operator.image.useDigest=false        --set preflight.image.repository=quay.io/cilium/cilium       --set preflight.image.useDigest=false       --set nodeinit.image.repository=quay.io/cilium/startup-script 
"cilium" has been added to your repositories
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "cilium" chart repository
Update Complete. ⎈Happy Helming!⎈
Error: values don't meet the specifications of the schema(s) in the following chart(s):
cilium:
- bpf.vlanBypass: Invalid type. Expected: [null,array], given: integer

make[2]: *** [Makefile:37: kind-init] Error 1
make[2]: Leaving directory '/home/runner/work/spiderpool/spiderpool/test'
make[1]: *** [Makefile:299: e2e_init] Error 2
make[1]: Leaving directory '/home/runner/work/spiderpool/spiderpool'
make: *** [Makefile:327: e2e_init_cilium_with_ebpf] Error 2
```

--- a/docs/usage/install/overlay/get-started-cilium-zh_cn.md
+++ b/docs/usage/install/overlay/get-started-cilium-zh_cn.md
@@ -18,9 +18,11 @@
 - 准备好一个 Kubernetes 集群
 - 安装 Cilium 作为集群的缺省 CNI。如果未安装，可参考 [官方文档](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/) 或使用以下命令安装:
 
+    > Cilium 在运行时会扫描可用的 VLAN 设备和标签，并将过滤所有未知流量。在一些跨 VLAN 场景下访问时，可能会出现通讯问题，请在安装时使用 `--set bpf.vlanBypass={0}` 允许所有的 VLAN 标记通过，更多细节参考 [Cilium VLAN 802.1q 支持](https://docs.cilium.io/en/stable/configuration/vlan-802.1q/#vlan-802-1q)。
+
     ```shell
     ~# helm repo add cilium https://helm.cilium.io/
-    ~# helm install cilium cilium/cilium -namespace kube-system
+    ~# helm install cilium cilium/cilium -namespace kube-system --set bpf.vlanBypass={0}
     ~# kubectl wait --for=condition=ready -l k8s-app=cilium pod -n kube-system
     ```
 

--- a/docs/usage/install/overlay/get-started-cilium.md
+++ b/docs/usage/install/overlay/get-started-cilium.md
@@ -18,9 +18,11 @@ This page showcases the utilization of `Spiderpool`, a comprehensive Underlay ne
 - A ready Kubernetes cluster.
 - Cilium has been already installed as the default CNI for your cluster. If it is not installed, please refer to [the official documentation](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/) or follow the commands below for installation:
 
+    > Cilium will scan available VLAN devices and tags when running, and will filter all unknown traffic. In some cross-VLAN scenarios,there may be communication problems. Please use `--set bpf.vlanBypass={0}` during installation to allow all VLAN tags to pass. For more details, refer to [Cilium VLAN 802.1q support](https://docs.cilium.io/en/stable/configuration/vlan-802.1q/#vlan-802-1q).
+
     ```shell
     ~# helm repo add cilium https://helm.cilium.io/
-    ~# helm install cilium cilium/cilium -namespace kube-system
+    ~# helm install cilium cilium/cilium -namespace kube-system --set bpf.vlanBypass={0}
     ~# kubectl wait --for=condition=ready -l k8s-app=cilium pod -n kube-system
     ```
 

--- a/test/scripts/install-default-cni.sh
+++ b/test/scripts/install-default-cni.sh
@@ -172,15 +172,15 @@ function install_cilium() {
       # k8sServiceHost api-server address
       # k8sServicePort api-service port
       # bpf.vlanBypass allow vlan traffic to pass
-      KUBE_PROXY_REPLACEMENT=disabled
+      KUBE_PROXY_REPLACEMENT=false
       if [ "$DISABLE_KUBE_PROXY" = "true" ]; then
-        KUBE_PROXY_REPLACEMENT=strict
+        KUBE_PROXY_REPLACEMENT=true
       fi
       CILIUM_HELM_OPTIONS=" --set cni.exclusive=false \
                             --set kubeProxyReplacement=${KUBE_PROXY_REPLACEMENT} \
                             --set k8sServiceHost=${E2E_CLUSTER_NAME}-control-plane \
                             --set k8sServicePort=6443 \
-                            --set bpf.vlanBypass=0 "
+                            --set bpf.vlanBypass={0} "
       case ${E2E_IP_FAMILY} in
         ipv4)
             CILIUM_HELM_OPTIONS+=" --set ipam.operator.clusterPoolIPv4PodCIDRList=${CILIUM_CLUSTER_POD_SUBNET_V4} \

